test/cbmc/proofs/parsing/ProcessIPPacket/Makefile.json

```diff
@@ -12,6 +12,7 @@
   ],
   "DEF":
   [
+    "ipconfigDRIVER_INCLUDED_TX_IP_CHECKSUM=1",
```

Comment:
This would reduce the coverage. I shall look at this. I would ideally keep this in two different configs and test both.
Will update shortly.

Reply:
I without this line, the CBMC proofs took forever, in the function that checks the checksum per software.
I tried to raise the loop limit to 100, but that did unfortunately never complete.

Reply:
Today I tested your branch "[fix_icmp_response_checksum](https://github.com/bernd-edlinger/FreeRTOS-Plus-TCP/tree/fix_icmp_response_checksum)", and played with the definition of `ipconfigFORCE_IP_DONT_FRAGMENT`. When it is defined as `1`, there will be a compilation error:
`ipFRAGMENT_FLAGS_DONT_FRAGMENT` undeclared, because my `ipconfigETHERNET_DRIVER_FILTERS_PACKETS` is non-zero, and so 
`ipFRAGMENT_FLAGS_DONT_FRAGMENT` is not defined, see [here](https://github.com/FreeRTOS/FreeRTOS-Plus-TCP/blob/2134672c4876e12a978feb69ab98bf8dbcb81917/include/FreeRTOS_IP_Private.h#L358).

I would propose to remove the `#if` in FreeRTOS_IP_Private.h :
~~~
-#if ( ipconfigETHERNET_DRIVER_FILTERS_PACKETS == 0 )
     #if ( ipconfigBYTE_ORDER == pdFREERTOS_LITTLE_ENDIAN )
         /* The bits in the two byte IP header field that make up the fragment offset value. */
         #define ipFRAGMENT_OFFSET_BIT_MASK         ( ( uint16_t ) 0xff1fU )
         <snip>
         #define ipFRAGMENT_FLAGS_MORE_FRAGMENTS    ( ( uint16_t ) 0x2000U )
     #endif /* ipconfigBYTE_ORDER */
-#endif /* ipconfigETHERNET_DRIVER_FILTERS_PACKETS */
~~~
Ping the device went perfectly in both directions, also a heavy flood-ping from a Linux device.

Thanks